java/dbschema/manifest.mf

```diff
@@ -1,6 +1,6 @@
 Manifest-Version: 1.0
 OpenIDE-Module: org.netbeans.modules.dbschema/1
-OpenIDE-Module-Implementation-Version: 4
+OpenIDE-Module-Implementation-Version: 5
```

Comment:
Here you probably want to increment `specification-version` (see `project.properties`). You're changing `friend-public` API in a binary-incompatible way (but that should not matter as all friends are recompiled in the tree - right @JaroslavTulach  ?)

Reply:
Thanks for CCing me, Sváťo!

I see that the [signature test](https://travis-ci.com/github/apache/netbeans/jobs/482358349) failed with:
```
Class org.netbeans.modules.dbschema.jdbcimpl.IndexElementImpl

  "E1.2 - API type removed" : method protected void org.netbeans.modules.dbschema.jdbcimpl.IndexElementImpl.initColumns(java.util.LinkedList)

Class org.netbeans.modules.dbschema.jdbcimpl.SchemaElementImpl

  "E1.2 - API type removed" : method public void org.netbeans.modules.dbschema.jdbcimpl.SchemaElementImpl.initTables(org.netbeans.modules.dbschema.jdbcimpl.ConnectionProvider,java.util.LinkedList,java.util.LinkedList)
```

I suggest to not change any public signature incompatibly.  While...

> changing friend-public API in a binary-incompatible way (but that should not matter as all friends are recompiled in the tree

...is possible. You'd have to increase dependencies of all friend modules. It is easier to keep `LinkedList` where it was. Even I agree with [code against interfaces not implementation](http://wiki.apidesign.org/wiki/Code_Against_Interfaces%2C_Not_Implementations) advice, I value backward compatibility more.